lib/bolt/inventory.rb

```diff
@@ -188,8 +188,15 @@ def update_target(target)
         data['config'] = {}
       end
 
-      unless data['config']['transport']
-        data['config']['transport'] = 'local' if target.name == 'localhost'
+      if target.name == 'localhost'
+        unless data['config']['transport']
+          data['config']['transport'] = 'local'
+        end
+        t = data['config']['transport']
```

Comment:
I don't think we should update data on other transports after seeing how this works it definitely feels wrong.